ibc-clients/cw-context/src/types/msgs.rs

```diff
@@ -0,0 +1,296 @@
+//! Defines the messages sent to the CosmWasm contract by the 08-wasm proxy
+//! light client.
+use std::str::FromStr;
+
+use cosmwasm_schema::cw_serde;
+use ibc_client_wasm_types::serializer::Base64;
+use ibc_core::client::types::proto::v1::Height as RawHeight;
+use ibc_core::client::types::Height;
+use ibc_core::commitment_types::commitment::{CommitmentPrefix, CommitmentProofBytes};
+use ibc_core::host::types::path::Path;
+use ibc_core::primitives::proto::Any;
+use prost::Message;
+
+use super::error::ContractError;
+
+pub type Bytes = Vec<u8>;
```

Comment:
Does it make sense to put it in some `utils` module? Or even move to `ibc_primitives` prelude?

Reply:
[6de21ed](https://github.com/cosmos/ibc-rs/pull/1167/commits/6de21edb62137b9d74daf1bcacf3b626e45013cc)